src/ioctls/vm.rs

```diff
@@ -608,6 +608,15 @@ impl AsRawFd for VmFd {
     }
 }
 
+impl Clone for VmFd {
```

Comment:
Conceptually I find it a bit weird to have a clone of a VmFd. What is the usecase for adding this? I am thinking that instead of having clones of VmFd you should be able to work with a reference to VmFd.

Reply:
Ok I have worked around the problem by using Arc<> from the code consuming the VmFd. There's no need for the library to handle the VmFd cloning.